service/src/main/resources/templates/ehr_immunization_observation_statement_template.mustache

```diff
@@ -14,9 +15,8 @@
             </pertinentAnnotation>
         </pertinentInformation>
         {{/pertinentInformation}}
-        {{{code}}}
         {{#participant}}
         {{{.}}}
         {{/participant}}
     </ObservationStatement>
-</component>
+</component>
```

Comment:
It's often considered good practice to include a final new line. That's partly why github gives the little warning when it's absent. Most (possibly all?) of the reasons why it's good practice don't apply to us though. One effect we *can* see is the `</component><component>` on one line in the output, where we might prefer it to be formatted as it was before.

Reply:
Added back final new line and updated tests 
Thanks for the suggestion 👍